Challenge.md

```diff
@@ -11,6 +11,17 @@
 
 2. 求基态简并度。 
 
+-----------------------------------------
+
+*translation*: In the Buckyball structure as shown in the figure, we attach an ising spin <img src="http://latex.codecogs.com/svg.latex?\sigma_i=\pm 1"></img> on each vertex.
+Two neighboring spins are correlated by an anti-ferromagnetic coupling of unit strength.
```

Comment:
The neighboring spins interact with an anti-ferromagnetic coupling of unit strength.